### ISSUE TITLE
error while compiling Alarm manager sample app

### ISSUE THREAD

**Issue by JanuszChmiel** [jmpessoa/lazandroidmodulewizard#66]
Finally here is detailed log about compilation. I AM helpless to find The cause myself. I have installed all SDK platform tools needed, including Android API 17. Unfortunately, more than 1 or 2 sample projects can not be compiled. I Am also using ANDROID NDK R10E. Or should I use lover supported version instead? 
I  know, that I AM bothering you but I can not find The cause myself even by reading log carefully.

**Comment by jmpessoa** [comment]
Hi, Janusz!

Sorry.... here the compilation and apk build/install/run work correctly!   

You are using command lines tools?

**Comment by JanuszChmiel** [comment]
No. I AM using build APk and run command. I AM very sad, that I do not 
know, how to solve this issue. I Am using Android NDK R10E. videoview 
example can be compiled without problems.
But many apps no.

**Comment by JanuszChmiel** [comment]
And I Am using 32 bit installations of JRE and JDK. Jre and JDK has 
version number u201.
I have also tried lover version of Lazandroid with Lamw from 2017, but 
also there some packages from demo folder can not be successfully compiled.
But to be also positive to you. This project has A real future and it is 
The best opensource Pascal project. Even anroid permissions confirmation 
is correctly pronounced by Talkback, very accessible environment.
I will try to create a new project from LAMW and I will try to manually 
import .pas file here to test, if I will be able to compile.
I will also try to create new projects and with some simple code. Empty 
form with only title and some Pascal code.
